bootstrap.ts

```diff
@@ -96,3 +95,13 @@ $injector.requireCommand("doctor", "./commands/doctor");
 
 $injector.require("utils", "./utils");
 $injector.require("bplistParser", "./bplist-parser");
+
+$injector.requirePublicClass("deviceEmitter", "./mobile/mobile-core/device-emitter");
+
+$injector.register("mobilePlatformsCapabilities", {});
+$injector.register("config", {});
```

Comment:
maybe it's better to register mock classes and require the files here instead of registering them